modules/generate-diagnostic-bundle-using-roxctl-cli.adoc

```diff
@@ -1,19 +1,64 @@
 // Module included in the following assemblies:
 //
 // * configuration/generate-diagnostic-bundle.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="generate-diagnostic-bundle-using-roxctl-cli_{context}"]
 = Generating a diagnostic bundle by using the roxctl CLI
 
 You can generate a diagnostic bundle by using the `roxctl` CLI.
 
+.Prerequisites
+* To generate a diagnostic bundle, you need `read` permission for the `Administration` resource. This is required for versions of the `DebugLogs` resource older than version 3.73.0.
+
+* You have configured the `ROX_PASSWORD` and the `ROX_CENTRAL_ADDRESS` environment variables.
+
+** Set the `ROX_PASSWORD` and the `ROX_CENTRAL_ADDRESS` environment variables:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_PASSWORD=_<rox_password>_ && export ROX_CENTRAL_ADDRESS=_<address>:<port_number>_ <1>
+----
++
+<1> For `<rox_password>`, specify the stackrox administration password.
```

Comment:
```suggestion
<1> For `<rox_password>`, specify the {product-title-short} administrator password.
```

Reply:
@rukletsov please verify that stackrox administration password = RHACS administrator password? (We seem to use simply "administrator password" in the current docs, but maybe adding RHACS makes it more clear.)

Reply:
Done